src/pcre2_jit_char_inc.h

```diff
@@ -639,7 +639,7 @@ if (category_list == UCPCAT_ALL)
 if (*cc != XCL_END)
   {
 #if defined SUPPORT_UNICODE && (PCRE2_CODE_UNIT_WIDTH == 8 || PCRE2_CODE_UNIT_WIDTH == 16)
-  if (common->utf && compares == 0 && (status & XCLASS_IS_ECLASS))
+  if (common->utf && compares == 0 && !(status & XCLASS_IS_ECLASS))
```

Comment:
It was a typo. Luckily, it has no other effect than a deoptimization (xclass case is not optimized, the eclass case always ignored the computed value). Fortunately I noticed it.